app/classifier/index.cjsx

```diff
@@ -49,10 +75,15 @@ Classifier = React.createClass
       @loadSubject subject
     if nextProps.classification isnt @props.classification
       @prepareToClassify nextProps.classification
+    @context.geordi?.remember subjectID: @props.subject?.id
+    @reCheckIfInterventionNeeded()
```

Comment:
Did you intend to have this check every time props is received? If the client is going to be in the app context, you can check if there is a change in context using `nextContext`. However, I don't think context should be used.